src/zeebe/util/ElementsUtil.js

```diff
@@ -0,0 +1,123 @@
+import { find, forEach, isArray } from 'min-dash';
```

Comment:
Correct me when I'm wrong, but some utils look like duplicated code. Any chance to have a `src/shared` folder to make this reusable across the platforms?

Reply:
The files are identical :)

Reply:
removed duplicated code in [44b8467](https://github.com/bpmn-io/extract-process-variables/pull/20/commits/44b84678ce58d15430c174fbdc572c7337073d5b)